v0/destinations/candu/data/CanduIdentify.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "destKey": "traits",
+    "sourceKeys": ["traits", "context.traits"],
+    "required": false
+  },
+  {
+    "destKey": "userId",
+    "sourceKeys": "userId",
+    "required": false
+  },
```

Comment:
we should use `getFieldValueFromMessage(message, "userIdOnly")` inorder to extract this as we have a genericFieldMapping for this which looks for `userId` in 5 different places.